### DISCUSSION TITLE
Accounts Icon

### DISCUSSION THREAD

**Discussion by amyodea** [twilio-labs/paste#3880] [Request: Icon] [answered]
### Description

Hey Team! I need to add a new icon for 'Accounts' and 'Account Dashboard' into the Paste library. Thank you! 

![Screenshot 2024-04-29 at 10 32 38](https://github.com/twilio-labs/paste/assets/104831341/0a493dbf-9259-4765-8e58-9c4b4d6ee14d)

![Screenshot 2024-04-29 at 10 33 49](https://github.com/twilio-labs/paste/assets/104831341/5a697777-27f3-4e10-b04d-55b0eb828a9a)

### Link to icon

https://www.streamlinehq.com/icons/streamline-light/programing-apps-websites/amazon-web-services/elastic-compute-cloud-instances

https://www.streamlinehq.com/icons/streamline-light/business-products/trading/trading-website-browser

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by serifluous** [comment]
Hey @amyodea ! Can you fill out a dependency request for these icons? https://airtable.com/appwvsQKyJHfnG5Sa/shrvMvL3BG6Zm0f9l

btw i've consolidated your 2 discussions, so we can just track the icons in this one.

For the Accounts icon:
1. We already have an icon called ProductAdminAccountsIcon. We can update the glyph in that for a new one since the current glyph conflicts with ProductSwitcherIcon anyway.
2. FYI the glyph you suggested looks pretty close to both CopyIcon and ProductEmailAPIIcon. I'll drop this note in Slack, too, since you polled the Comms designers, if you want more opinions.

<img width="202" alt="Screenshot 2024-04-29 at 11 50 14 AM" src="https://github.com/twilio-labs/paste/assets/3821840/146745dd-a0ef-40e3-923f-df33e2cec439">
<img width="227" alt="Screenshot 2024-04-29 at 11 45 58 AM" src="https://github.com/twilio-labs/paste/assets/3821840/24c95f29-a89c-4566-9b8c-35232180fbf8">

**Reply by amyodea** [reply]
Great, thanks for that Sarah! Yeah, responded in Slack to get thoughts from the other designers too. I don't anticipate them being used in close context. Will file the dependency after I get thoughts from the team!

**Comment by nkrantz** [comment, marked as answer]
Hey @amyodea - the icons are now available!
